### PR TITLE
Minor cleanup of random things

### PR DESCRIPTION
**Pull request by aschleck**
I'm in the market for something fun other than feature tweaking, let me know if you have something in mind :)

Some things on my mind:
* move ./data/AllData.json and ./data/county_gps.json into public to be fetched rather than directly embedded in the JS bundle (would moving these mess up your pipelines? do I just update normalize_data.js?)
* move files into organized directories
* seeing what can be done about MyRoutes and the history stuff in MyTabs
* playing around with those UI mocks? probably a bad idea

### CODE REVIEW

File: website/src/Hospitals.js [hunk 1]
```diff
@@ -40,10 +24,11 @@ const DetailCaseListWidget = (props) => {
         <>
             <Typography variant="body1">
                 Please follow
-                <Link className={classes.link} href="https://www.cdc.gov/coronavirus/2019-ncov/if-you-are-sick/steps-when-sick.html">
+                {' '}
+                <Link href="https://www.cdc.gov/coronavirus/2019-ncov/if-you-are-sick/steps-when-sick.html">
                     CDC guidance.
                 </Link>
-
+                {' '}
```
Comment (xunhuang) [inline website/src/Hospitals.js:31]: you have no idea how much struggle I had with this .... :)

File: website/src/Theme.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+            },
+        },
+        MuiToggleButton: {
+            sizeSmall: {
+                //This can be referred from Material UI API documentation.
+                maxHeight: 24,
+            }
+        }
+    },
+});
```
Comment (xunhuang) [inline website/src/Theme.js:17]: thank you for extracting this out!

File: website/src/Header.js [hunk 3]
```diff
@@ -236,42 +236,22 @@ const DonateButton = (props) => {
...
+                [PinterestShareButton, PinterestIcon],
+                [EmailShareButton, EmailIcon],
+            ].map(([Button, Icon]) => (
+                <Button url={props.url} quote={props.quote}>
+                  <Icon size={32} round={true} />
+                </Button>
+            ))}
+        </div>
+    );
 }
```
Comment (xunhuang) [inline website/src/Header.js:255]: love this!

File: website/src/GeoLocation.js [hunk 4]
```diff
@@ -16,50 +16,46 @@ async function fetchCounty() {
...
-    if (!location) {
-        location = {
+    const methods = [
+        () => fetchApproxIPLocationIPDataCo(firebaseConfig.ipdataco_key),
+        () => fetchApproxIPLocationIPDataCo(firebaseConfig.ipdataco_key2),
+        () => fetchApproxIPLocationIPDataCo(firebaseConfig.ipdataco_key3),
+        () => fetchApproxIPLocationIPGEOLOCATION(),
+        () => fetchApproxIPLocationGoogle(),
+        () => {
+            logger.logEvent("LocationNoFoundAfterAPI");
```
Comment (xunhuang) [inline website/src/GeoLocation.js:26]: I like this way of supplying the default at the end.

File: website/src/Theme.js (hunk 2 above)
Comment (xunhuang) [reply website/src/Theme.js:17]: I love your cleanup! really like how your make local refactoring and make each piece code look great and honoring the logic that was there before, just re-expressing them in a way that's easy to understand. quite a talent!  (at least my messy code is quite localized.)

**Comment by xunhuang** [comment]
> move ./data/AllData.json and ./data/county_gps.json into public to be fetched rather than directly embedded in the JS bundle (would moving these mess up your pipelines? do I just update normalize_data.js?)

if you run normalize_data.js and change where the file goes, and read them back it's going to be ok. I think. That said, perf wise, is fetching from the public that different from being in the bundle since the app really needs the file to run anyways. I don't have big objection, tiny concerned that the two step will serialized (load the bundle, execute the bundle, then fetch). However, one benefit is once the file is in the public html, we can point whoever that wants to use it to download it, instead of directing them to github.

> move files into organized directories

Feel free! 

> seeing what can be done about MyRoutes and the history stuff in MyTabs

That is a mess. And I want introduce a third level tab, like those in the Map. would be great to have them work out the history stuff too. 

> playing around with those UI mocks? probably a bad idea

I think overall, we should think about how we should structure the UX. As it, things are getting very crowded and I'm out of my depth on how to organize things. Since you seem to have sharp clarity of organization, feel free to take a stab at it. I have 3 or 4 more graphs/map that I want to stick it in there somehow. :) 

Lastly thank you for your help on this! Other than myself, you have contributed to most to the project thus far, by far! it's our project now (not just mine.)  I'll add you as an admin to the firebase project.

**Comment by aschleck** [comment]
Wow thank you for such kind feedback! I am excited to join the Firebase project and see the hospitals tab finally load on my computer :D 

I will see what I can come up with for those other proposals. I really admire how far (and successfully) you've been pushing this project, it's nice to work with you!

**Comment by xunhuang** [comment]
I'm like a jack of all trades but master of none. I used to be a startup
guy but in virtualization technology (10+ years ago) but got suckered in
management along the way. Now I'm trying things out in consumer space,
don't really know anything, but willing to learn and try things out. While
I haven't learned much technical things at Facebook in my pointy-hair
management/sr engineer role, I appreciate the move fast and break things
culture. After all, life won't be lost if the dashboard is down/slightly
inaccurate, so why not run fast. I know I'm sloppy, but you should have
seen the code before I finally buckled down and came up with data layer
with AllData.json. Before, all the shitting logic in normalize_data.js was
in the frontend code, I was literally getting dizzy trying fix all the data
error in the UI code.  If I didn't rewrite at least that part before you
showed up, you would probably take a peak and say "shit code, I won't touch
that." At least now it's structurally sound, piecemeal bad :)

**Pull request merged**